nfs/server.sls

```diff
@@ -1,17 +1,33 @@
 {% from "nfs/map.jinja" import nfs with context %}
 
+{% if nfs.pkgs_server %}
 nfs-server-deps:
     pkg.installed:
         - pkgs: {{ nfs.pkgs_server|json }}
+{% endif %}          
 
 /etc/exports:
```

Comment:
Minor comment - consider using (say) `nfs-exports-configure` ID here instead of `/etc/exports:` - its more descriptive & less risk of ID collision on filename.